thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.media
+namespace java com.gu.contentatom.thrift.atom.media
+#@namespace scala com.gu.contentatom.thrift.atom.media
+
+typedef i64 Version
+
+enum AssetType {
+  YOUTUBE,
+  FACEBOOK,
+  PODCAST,
+  DAILYMOTION,
+  MAINSTREAM,
+  SUBTITLES
+}
```

Comment:
Could we add something to this list like `URI`, which can accept URIs?
This would be useful for the complete-backup system... or at least I think it might.

It also means we could test encodings from pluto asset URLs?